DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/PresetServlet.java

```diff
@@ -34,39 +34,39 @@
-        /*
-      // parse each parameter from the save preset form.
-      String userEmail = request.getParameter("userEmail");
-      String userId = request.getParameter("userId");
-      String presetId = request.getParameter("presetId");
-      String simName = request.getParameter("sim_name");
-      String fromDate = request.getParameter("start_date");
-      String toDate = request.getParameter("end_date");
-      double dailyBudget = Double.parseDouble(request.getParameter("daily_budget"));
-      String location = request.getParameter("location");
-      String language = request.getParameter("language");
-      String domain = request.getParameter("domain");
-      String target = request.getParameter("target_page");
-      String adText = request.getParameter("ad_text");
-      double cpc = Double.parseDouble(request.getParameter("cpc"));
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    /*
+    // parse each parameter from the save preset form.
+    String userEmail = request.getParameter("userEmail");
+    String userId = request.getParameter("userId");
+    String presetId = request.getParameter("presetId");
+    String simName = request.getParameter("sim_name");
+    String fromDate = request.getParameter("start_date");
+    String toDate = request.getParameter("end_date");
+    double dailyBudget = Double.parseDouble(request.getParameter("daily_budget"));
+    String location = request.getParameter("location");
+    String language = request.getParameter("language");
+    String domain = request.getParameter("domain");
+    String target = request.getParameter("target_page");
+    String adText = request.getParameter("ad_text");
+    double cpc = Double.parseDouble(request.getParameter("cpc"));
 
-      Entity presetEntity = new Entity("PresetData");      
-      presetEntity.setProperty("userEmail", userEmail);
-      presetEntity.setProperty("presetId", presetId);
-      DSACampaign dsaCampaign = new DSACampaign(0, userId, 0, simName, fromDate, toDate, dailyBudget, location, domain, target);
-      Gson gson = new Gson();
-      String dsaCampaignData = gson.toJson(dsaCampaign);
-      presetEntity.setProperty("presetData", dsaCampaignData);
-      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-      datastore.put(presetEntity);
-      */
-    }
+    Entity presetEntity = new Entity("PresetData");      
+    presetEntity.setProperty("userEmail", userEmail);
+    presetEntity.setProperty("presetId", presetId);
+    DSACampaign dsaCampaign = new DSACampaign(0, userId, 0, simName, fromDate, toDate, dailyBudget, location, domain, target);
+    Gson gson = new Gson();
+    String dsaCampaignData = gson.toJson(dsaCampaign);
+    presetEntity.setProperty("presetData", dsaCampaignData);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(presetEntity);
+    */
```

Comment:
Why is this all being commented out? If you don't want it, you should remove it.

Reply:
I want that code to be there, just waiting for the settings to be finalized.

Reply:
I see. But in general we don't want to have commented code chunks that aren't immediately part of the implementation. I think it would be better to remove it in this PR and then add the correct implementation in the next PR. Also I think a lot of this code can be combined into a CampaignSettings class that each class uses to store/interact with the data, which could make this code redundant once settings are finalized.

Reply:
Okay, I'll remove it for this PR and create a class for the settings data.